src/olympia/migrations/990-add-private-whiteboard.sql

```diff
@@ -0,0 +1,7 @@
+CREATE TABLE `private_whiteboard` (
+    `created` datetime(6) NOT NULL,
+    `modified` datetime(6) NOT NULL,
+    `addon_id` integer NOT NULL PRIMARY KEY,
```

Comment:
This is not your fault, you did everything right, but this will fail on dev/stage/prod, because they unfortunately have a different db schema for most of the `ids`: they are `unsigned`. So the FK constraint will complain.

Can you add a comment at the top like this:
`-- Note: if the migration fails for you locally, remove the 'UNSIGNED' next to addon_id below.`
And modify `addon_id` to be `UNSIGNED`.